en/docs/visual-programming/sequence-diagram-view.md

```diff
@@ -1,28 +1,55 @@
-# Sequence diagram view
-With Ballerina, you can view your code in a graphical mode and develop your program logic through it. This
-allows you to look at your program logic holistically and gives you the ability to comprehend the program at 
-a glance. This visual representation has its roots in sequence diagrams and can capture how the logic of
-the program flows, how the concurrent execution flow works, which remote endpoints are involved, and how they interact 
-with the different objects in the program.
+# Sequence Diagram view
+
+When you use the Ballerina VS Code extension, you can view your code in graphical mode and develop your program logic through it. This
+allows you to look at your program logic holistically and comprehend the program at a glance. 
+
+This visual representation has its roots in sequence diagrams and can capture how the logic of your program flows, how the concurrent execution flow works, which remote endpoints are involved, and how those endpoints interact with the different objects in the program.
 
 ## Switch to graphical mode
+
 There are two methods for switching to the graphical mode using the VS Code extension.
 
-1. Execute the `Ballerina: Show Diagram` command from the command palette.
+-   Execute the `Ballerina: Show Diagram` command from the command palette.
+    
     ![command screenshot](../img/show-diagram-command-palette.png)
 
-2. Click the `Show Diagram` button in the editor group.
+-   Click `Show Diagram` in the editor group.
+    
     ![button screenshot](../img/editor-group-button.png)
 
+Shown below is the code editor and graphical editor in view.
+
+![graphical view](../img/graphical-view.png)
+
```

Comment:
Shall we add a section here to explain the following options? I don't think it is explained anywhere. Will it be too much info. wdyt? @anupama-pathirage @praneesha 

<img width="1219" alt="Screenshot 2023-01-16 at 15 02 02" src="https://user-images.githubusercontent.com/21237558/212644927-72ba498e-bf39-4897-b52f-0055b12acd9b.png">

Reply:
IMO, that is too much and all the terms are self explanatory as those are coming from the language itself.

Reply:
ack